### PR TITLE
Core work for filter/drop down component created component

### PR DESCRIPTION
**Pull request by GraysonQuartech**

### CODE REVIEW

File: src/components/Filter.tsx [hunk 1]
```diff
@@ -1,9 +1,54 @@
...
+ */
+const Filter = (props: filterProps) => {
+  const [taxon, setTaxon] = React.useState("");
+
+  const handleChange = (event: SelectChangeEvent) => {
+    setTaxon(event.target.value as string);
+  };
+
+  return (
+    <Box sx={{ minWidth: 120 }}>
```
Comment (MacQSL) [inline src/components/Filter.tsx:33]: useStyles hook pls

File: src/components/FilterRow.tsx [hunk 2]
```diff
@@ -1,10 +1,20 @@
+//React packages and components
 import React from "react";
 import Filter from "./Filter";
+//Constants + Data
+import { testArray } from "../utils/constants";
 
 function FilterRow() {
```
Comment (MacQSL) [inline src/components/FilterRow.tsx:7]: arrow function / return type

File: src/components/FilterRow.tsx [hunk 3]
```diff
@@ -1,10 +1,20 @@
...
   return (
     <div>
-      <Filter />
+      <Filter classificationLevel={"Kingdom"} dropDownTaxons={testArray} />
```
Comment (MacQSL) [inline src/components/FilterRow.tsx:10]: map these values from the data set

File: src/components/Filter.tsx [hunk 4]
```diff
@@ -1,9 +1,54 @@
...
 
+/*
+ * Main component Function.
+ * This component is a basic MUI select
+ * It receives an array of taxons for the user to select from
+ */
+const Filter = (props: filterProps) => {
+  const [taxon, setTaxon] = React.useState("");
+
+  const handleChange = (event: SelectChangeEvent) => {
```
Comment (MacQSL) [inline src/components/Filter.tsx:28]: return type

File: src/components/Filter.tsx (hunk 1 above)
Comment (GraysonQuartech) [reply src/components/Filter.tsx:33]: Removed style line for now. Will add again in next PR using style hook.

**Pull request merged**